src/batched/dense/KokkosBatched_Gemm_Decl.hpp

```diff
@@ -227,10 +227,7 @@ class BatchedSerialGemm;
 ///                             BoundsCheck::No  The functor will NOT perform bound checks
 /// \tparam ArgAlphaFmaTag      Specifies whether to apply alpha during fmas.
 ///                             AlphaFmaTag::Yes alpha will be applied during fma (C = C * alpha + AB).
-///                             AlphaFmaTag::No  alpha will not be applied during fma (C = C + AB).
-/// \tparam ArgAlphaMulTag      Specifies whether to perform global memory access
-///                             AlphaFmaTag::Yes alpha will be applied during mul (A * B * alpha).
-///                             AlphaFmaTag::No  alpha will not be applied during mul (A * B).
+///                             AlphaFmaTag::No  alpha will not be applied during mul (A * B * alpha).
```

Comment:
It should be "alpha will be applied during mul"

Reply:
Done.

Reply:
Thanks, Evan